iota-client/src/client.rs

```diff
@@ -114,6 +114,10 @@ pub struct BrokerOptions {
     pub(crate) automatic_disconnect: bool,
     #[serde(default = "default_broker_timeout")]
     pub(crate) timeout: Duration,
+    #[serde(default = "default_broker_use_ws")]
```

Comment:
We should also use `rename = "defaultBrokerUseWs"` here, because in nodejs it's expected to be camelCase